common/religion/religions/naarism.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_crime
 	doctrine = doctrine_witchcraft_crime
 
+	#Magic
+	doctrine = doctrine_light_magic_approved
+	doctrine = doctrine_shadow_magic_crime
+	doctrine = doctrine_disorder_magic_crime
+	doctrine = doctrine_order_magic_accepted
+	doctrine = doctrine_life_magic_ignored
+	doctrine = doctrine_death_magic_crime
+	doctrine = doctrine_elemental_spirit_magic_shunned
```

Comment:
Naarism has came far to accept elemental magic on Outland, it should definitely be accepted.

Reply:
I was thinking of this thing

![image](https://user-images.githubusercontent.com/46576860/107844370-ca104600-6de3-11eb-80bc-bb1d0b346bdc.png)

<https://wow.gamepedia.com/Heretic>

<https://wow.gamepedia.com/Draenei_Artificer#Quotes>

Reply:
Such prejudice does still exist, however Velen as the religious head of Naarism considered them as legitimate.

>Gossip What did you do then?
I returned to my people and was greeted with smiling faces and outstretched arms, mostly from the Broken. The prejudices against us from many draenei had continued in my absence. But it was as if they knew that I was coming, and I soon found out why.
Velen, the prophet and leader of all of the draenei, had seen my return in one of his visions and was there to greet me as well. He asked to speak with me in private and as I related my story to him, he nodded and smiled, for this too he had foreseen.
Gossip Please, go on.
Velen urged me to take a leadership role amongst the Broken. But more importantly, he asked me to teach shamanism to all; Broken and draenei alike. I accepted.
We knew then that given the prejudices of some that this would be a difficult proposition at best. There are those that despise the Broken because we are a frightening reminder of the past. And there are others who believe the way of the shaman to be a blasphemy against the Light.
In any case, that is my tale.

https://wow.gamepedia.com/Farseer_Nobundo

Reply:
Ok and done